net/samba4/Portfile

```diff
@@ -78,19 +76,17 @@ patchfiles-append   patch-ldb-test.diff
 # https://bugzilla.samba.org/show_bug.cgi?id=15850
 patchfiles-append   patch-util-crypt.diff
 
-configure.perl      ${perl5.bin}
-configure.python    ${prefix}/bin/python${py_ver}
-configure.env-append \
-                    YAPP=${prefix}/bin/yapp-${perl5.major}
+configure.perl          ${perl5.bin}
+configure.python        ${prefix}/bin/python${py_ver}
+configure.env-append    YAPP=${prefix}/bin/yapp-${perl5.major}
+configure.args-append   --enable-fhs
 
 platform darwin {
-    configure.args-append   --without-acl-support
+    configure.args-append \
+        --without-acl-support \
+        --disable-avahi
```

Comment:
Move this outside the platform block as well.

Reply:
Hmm, I think this was first put inside a `darwin` block by you. I know Samba currently doesn't support ACL on Darwin (I suppose this means file system-level ACL integration, so userspace ACL via VFS should work - I think this flag only affects AD DC and Ceph functionalities), so this flag is required on macOS/Darwin let Samba build at all. Also, `avahi` makes sense if there is no native mDNS support.

What was the point of this block originally? To support FreeBSD? I saw there are some ports in this repo with `freebsd` as a target platform. In that case, it totally makes sense to have these two flags in this block.

Reply:
I just meant the avahi one - we do not selectively remove the dependency and therefore we shouldn't selectively add the configure flag. The ACL one was added because the build fails otherwise. It's in a platform block to make it clear that this is macOS specific. If we want to allow avahi support on some platforms then it would be a variant that might be enabled by default on those platforms. You can implement that if you want, but not required.

The port doesn't currently support FreeBSD or Linux since we haven't added that to `platforms`. But, it doesn't hurt to be accurate.

Reply:
Changes look good, but I think you missed moving this next to `--enable-fhs`.